src/pyclass_init.rs

```diff
@@ -183,7 +183,7 @@ impl<T: PyClass> PyClassInitializer<T> {
         unsafe { self.create_cell_from_subtype(py, T::type_object_raw(py)) }
     }
 
-    /// Create a new PyCell and initialize it given a typeobject `subtype`.
+    /// Creates a new PyCell and initializes it given a typeobject `subtype`.
     /// Called by our `tp_new` generated by the `#[new]` attribute.
```

Comment:
Similarly would be nice to remove "our" here:

```suggestion
    /// Called by the Python `tp_new` implementation generated by a `#[new]` function in a `#[pymethods]` block.
```